.github/ISSUE_TEMPLATE/---documentation-report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "\U0001F4D8 Documentation report"
+about: 'Something isn''t described correctly in the documentation or needs to be updated?
```

Comment:
Is it better to use double quotes here to avoid escaping?

Reply:
@marksweb Done! 😀